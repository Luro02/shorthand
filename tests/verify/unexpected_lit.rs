use shorthand::ShortHand;

#[derive(ShortHand)]
#[shorthand(verify(fn = b"Self::verify_field"))]
struct Example {
    field: usize,
}

fn main() {}
