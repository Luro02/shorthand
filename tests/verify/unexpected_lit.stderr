error: unexpected literal type `byte string`
 --> $DIR/unexpected_lit.rs:4:25
  |
4 | #[shorthand(verify(fn = b"Self::verify_field"))]
  |                         ^^^^^^^^^^^^^^^^^^^^^
